src/app/accounts/reducer.ts

```diff
@@ -8,12 +9,25 @@ import * as actions from './actions';
 import { DestinyAccount } from './destiny-account';
 
 export interface AccountsState {
+  /**
+   * A list of all accounts loaded from Bungie.net.
+   */
   readonly accounts: readonly DestinyAccount[];
-  // TODO: just the ID?
-  readonly currentAccount: number;
+  /**
+   * Platform Membership ID of the currently selected account. This may not
+   * correspond to any account in the accounts array!
+   */
+  readonly currentAccountMembershipId: string | undefined;
+  /** Destiny version of the currently selected account */
+  readonly currentAccountDestinyVersion: DestinyVersion | undefined;
```

Comment:
I'd tried to be clever before by storing an index into `accounts` but it turns out it's much better to just directly store the key you want to use.